sig-feature-store/README.md

```diff
@@ -0,0 +1,51 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# Feature Store Special Interest Group
+
+SIG Feature Store covers the definition, management, storage, discovery, and serving of features to models.
+
+The [charter](charter.md) defines the scope and governance of the Feature Store Special Interest Group.
+
+## Meetings
+* Monthly SIG Meeting: [Wednesdays at 18:00 PT (Pacific Time)]() (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1h2BMuN6V6cGFQrSRk7msmYVmZdrouLRV_EPR_tphZrU/edit#).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Oleksii Moskalenko (**[@pyalex](https://github.com/pyalex)**), Gojek
+* Willem Pienaar (**[@woop](https://github.com/woop)**), Gojek
+
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Oleksii Moskalenko (**[@pyalex](https://github.com/pyalex)**), Gojek
+* Willem Pienaar (**[@woop](https://github.com/woop)**), Gojek
+
+## Contact
+- Slack: [#feast](https://kubernetes.slack.com/messages/feast)
```

Comment:
@woop Slack channel doesn't look right; Do you mean kubeflow?

Reply:
Good catch. Fixed.